src/helpers/repoutils.ts

```diff
@@ -15,15 +30,15 @@ export class RepoUtils {
 
     //Checks to ensure it's a Team Foundation Git repo, then ensures it's hosted on visualstudio.com
     public static IsTeamFoundationServicesRepo(url: string): boolean {
-        if (RepoUtils.IsTeamFoundationGitRepo(url) && url.toLowerCase().indexOf(".visualstudio.com") >= 0) {
+        if (url.toLowerCase().indexOf(".visualstudio.com") >= 0) {
             return true;
         }
         return false;
     }
 
-    //Checks to ensure it's a Team Foundation repo, then ensures it's *not* on visualstudio.com
+    //This is the "catch all" method.  A repository hosted on TFS is defined by not being on "visualstudio.com"
     public static IsTeamFoundationServerRepo(url: string): boolean {
-        if (RepoUtils.IsTeamFoundationGitRepo(url) && !RepoUtils.IsTeamFoundationServicesRepo(url)) {
+        if (!RepoUtils.IsTeamFoundationServicesRepo(url)) {
```

Comment:
At this iteration, I assume that the url that I'm checking has already been verified as a valid url.  That said, I haven't gotten to the "possibly" scenario yet (which I believe is multiple collections).  Right now, the code will do a Team Services repository and, presumably, a "possibly" scenario is for on-prem.  Once I get to doing on-prem scenarios, I expect this to become clearer.